libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -131,6 +132,14 @@ void AP_Mount::init()
             _num_instances++;
             break;
 #endif // HAL_MOUNT_SCRIPTING_ENABLED
+
+#if HAL_MOUNT_XACTI_ENABLED
+        // check for Xacti gimbal
+        case Type::Xacti:
+            _backends[instance] = new AP_Mount_Xacti(*this, _params[instance], instance);
```

Comment:
check alloc failure?

Reply:
I think this is a nice-to-have so I will mark it as resolved.